hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/volume/MutableVolumeSet.java

```diff
@@ -475,7 +475,7 @@ public double getIdealUsage() {
     for (StorageVolume volume: volumeMap.values()) {
       SpaceUsageSource usage = volume.getCurrentUsage();
       totalCapacity += usage.getCapacity();
-      totalUsed += usage.getUsedSpace();
+      totalUsed += (usage.getCapacity() - usage.getAvailable());
```

Comment:
We can change this to totalFree, and use "(totalCapacity - totalFree) /  totalCapacity"  to save some computations.

Reply:
Same for DiskBalancerService.java